projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx

```diff
@@ -95,86 +98,123 @@ function SubscriptionsSettings( props ) {
-								'jetpack'
-							) }
-							{ isBlockTheme && subscribeModalEditorUrl && (
-								<>
-									{ ' ' }
-									<ExternalLink href={ subscribeModalEditorUrl }>
-										{ __( 'Preview and edit the pop-up', 'jetpack' ) }
-									</ExternalLink>
-								</>
-							) }
-						</p>
-					</FormFieldset>
-				}
-			</SettingsGroup>
-			{ getSubClickableCard() }
+					<ModuleToggle
+						slug="subscriptions"
+						disabled={ unavailableInOfflineMode }
+						activated={ isSubscriptionsActive }
+						toggling={ isSavingAnyOption( 'subscriptions' ) }
+						toggleModule={ toggleModuleNow }
+					>
+						<span className="jp-form-toggle-explanation">{ subscriptions.description }</span>
+					</ModuleToggle>
+					{
+						<FormFieldset>
+							<ToggleControl
+								checked={ isSubscriptionsActive && isStbEnabled }
+								disabled={ isDisabled }
+								toggling={ isSavingAnyOption( [ 'stb_enabled' ] ) }
+								onChange={ handleSubscribeToBlogToggleChange }
+								label={ __(
+									'Enable the “subscribe to site” option on your comment form',
+									'jetpack'
+								) }
+							/>
+							<ToggleControl
+								checked={ isSubscriptionsActive && isStcEnabled }
+								disabled={ isDisabled }
+								toggling={ isSavingAnyOption( [ 'stc_enabled' ] ) }
+								onChange={ handleSubscribeToCommentToggleChange }
+								label={ __(
+									'Enable the “subscribe to comments” option on your comment form',
+									'jetpack'
+								) }
+							/>
+							<ToggleControl
+								checked={ isSubscriptionsActive && isSmEnabled }
+								disabled={ isDisabled }
+								toggling={ isSavingAnyOption( [ 'sm_enabled' ] ) }
+								onChange={ handleSubscribeModalToggleChange }
+								label={ __( 'Enable subscription pop-up', 'jetpack' ) }
+							/>
+							<p className="jp-form-setting-explanation">
+								{ __(
+									'Automatically add a subscription form pop-up to every post and turn visitors into subscribers. It will appear as readers scroll through your posts.',
+									'jetpack'
+								) }
+								{ isBlockTheme && subscribeModalEditorUrl && (
+									<>
+										{ ' ' }
+										<ExternalLink href={ subscribeModalEditorUrl }>
+											{ __( 'Preview and edit the pop-up', 'jetpack' ) }
+										</ExternalLink>
+									</>
+								) }
+							</p>
+						</FormFieldset>
+					}
+				</SettingsGroup>
+				{ getSubClickableCard() }
 
-			{ ! isLinked && ! isOffline && (
-				<ConnectUserBar
-					feature="subscriptions"
-					featureLabel={ __( 'Newsletter', 'jetpack' ) }
-					text={ __( 'Connect to manage your subscriptions settings.', 'jetpack' ) }
-				/>
-			) }
-		</SettingsCard>
+				{ ! isLinked && ! isOffline && (
+					<ConnectUserBar
+						feature="subscriptions"
+						featureLabel={ __( 'Newsletter', 'jetpack' ) }
+						text={ __( 'Connect to manage your subscriptions settings.', 'jetpack' ) }
+					/>
+				) }
+			</SettingsCard>
+			<SettingsCard
```

Comment:
@JuanLucha , can you create a follow up PR to separate out the "Messages" into its own component?

Reply:
Sure 👍